app/models/user.rb

```diff
@@ -14,11 +21,14 @@ class User < ApplicationRecord
   validates :password, presence: true,
     length: {minimum: Settings.password.min_length},
     allow_nil: true
+  validates :role_id, presence: true, allow_nil: true
```

Comment:
e nên validate như này
```suggestion
  validates :role_id, inclusion: {in: role_ids.keys}
```
và đặt default cho nó ở migration.
tham khảo bài này để hiểu hơn nhé :))
https://viblo.asia/p/enum-trong-rails-con-dao-hai-luoi-nwmGyElDGoW

Reply:
Cảm ơn anh.